CHANGELOG.md

```diff
@@ -9,6 +9,20 @@
 * Podspec-based documentation respects the `swift-version` config option.  
   [Orta Therox](https://github.com/orta)
 
+## 0.7.23
```

Comment:
Hit a snag with DangerCI requesting a CHANGELOG, should I not be including these changes?